### DISCUSSION TITLE
Having more controle over the headings style

### DISCUSSION THREAD

**Discussion by OLH21** [Bluemoondragon07/obsidian-big-and-bold#12] [Ideas]
The present Heading choice does not make à lot of diffrence between H2 and H3, H4 ..., 
It would be nice to have a little more controle over the heading size for each heading and maybe also some control on the padding before and after them. Every one has his own preferences over this question and the readibility.

thanks again for this beautiful and usefull theme :)

**Comment by Bluemoondragon07** [comment]
That's a good idea, and I will consider adding more control over headings in the next update. I'll probably start simple with things like alignment, size, and font, but I plan to eventually add finer control like borders, margins, and padding. Thanks for the suggestion!

**Reply by OLH21** [reply]
I'd really have more control over spacing between titles and also title and paragraphe,. I cannot figure how to do that with a snippet , i cannnot figure the right select /class to target :) and for the moment spacing is huge if you have outilnes with several titles following each others
![image](https://github.com/Bluemoondragon07/obsidian-big-and-bold/assets/47591060/128a9ab9-d86b-468e-a039-3ec2972e1a22)

**Reply by yarub123** [reply]
@Bluemoondragon07 Since we're on this topic, I think more users would enjoy having the ability to change the color of bold text. I was surprised this theme didn't have it.
